src/main.cpp

```diff
@@ -1,4 +1,5 @@
 #include "../inc/source_code.hpp"
+#include "../inc/type.hpp"
 #include "../inc/parser.hpp"
```

Comment:
```suggestion
#include "source_code.hpp"
#include "type.hpp"
#include "parser.hpp"
```

CMake's `include_directories()` takes care of this for us https://cmake.org/cmake/help/latest/command/include_directories.html